<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <!-- First publish_pointcloud node -->
  <node name="publish_pointcloud1" pkg="publish_pointcloud" type="publish_pointcloud">

	<param name="path" value="$(find publish_pointcloud)/data/office5.pcd" type="str" />
	<param name="frame_id" value="camera" type="str" />
	<param name="topic1" value="/pointcloud/output1" type="str" />
	<param name="hz" value="2" type="int" />

  </node>

  <!-- Second publish_pointcloud node -->
  <node name="publish_pointcloud2" pkg="publish_pointcloud" type="publish_pointcloud">

    <param name="path" value="$(find publish_pointcloud)/data/radiation_results.pcd" type="str" /> <!-- Change file path -->
    <param name="frame_id" value="camera" type="str" />
    <param name="topic2" value="/pointcloud/output2" type="str" /> <!-- Change topic name -->
    <param name="hz" value="2" type="int" />

  </node>
  <!-- Load ocotmap launch -->
  <include file="$(find publish_pointcloud)/launch/octomaptransform.launch" />

  <!-- RViz -->
  <node pkg="rviz" type="rviz" name="$(anon rviz)" respawn="false" output="screen" args="-d $(find publish_pointcloud)/rviz/OctomapShow.rviz"/>

</launch>
